src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -236,7 +260,9 @@ var EventPluginRegistry = {
    * @return {?object} The plugin that created the supplied event.
    * @internal
    */
-  getPluginModuleForEvent: function(event) {
+  getPluginModuleForEvent: function(
+    event: ReactSyntheticEvent
+  ): null | PluginModule {
```

Comment:
[Discussion] it feels weird that null is before the actual type, I would have written it 

```js
PluginModule | null
```

I don't mind it that way but I'd love to know if there's thoughts behind it